client/src/main/java/org/apache/uniffle/client/util/RssClientConfig.java

```diff
@@ -56,6 +58,9 @@ public class RssClientConfig {
   // When the size of read buffer reaches the half of JVM region (i.e., 32m),
   // it will incur humongous allocation, so we set it to 14m.
   public static String RSS_CLIENT_READ_BUFFER_SIZE_DEFAULT_VALUE = "14m";
+  // The tags specified by rss client to determine shuffle data placement.
+  public static String RSS_CLIENT_DATA_PLACEMENT_TAGS = "rss.client.data.placement.tags";
+  public static String RSS_CLIENT_DATA_PLACEMENT_TAGS_DEFAULT_VALUES = Constants.SHUFFLE_SERVER_VERSION;
```

Comment:
A shuffle server can have multiple tags. The version tag shouldn't be overrided the configurable tags.